tools/clang/tools/dxcompiler/dxcpdbutils.cpp

```diff
@@ -546,6 +541,8 @@ struct DxcPdbUtils : public IDxcPdbUtils, public IDxcPixDxilDebugInfoFactory
       } break; // hlsl::DFCC_ShaderDebugInfoDXIL
       } // switch (four_cc)
     } // For each part
+    
+    SetEntryPointToDefaultIfEmpty();
```

Comment:
It's possible `HandleDxilContainer` didn't find anything at all, and then need to look in the debug DXIL module. Better to call this at the end of the `try` block in DxcPdbUtils::Load